one-ui/ui/src/components/entities/entity-tiles.tsx

```diff
@@ -95,10 +126,13 @@ const EntityTiles: React.FC = () => {
             output = <div className={styles.cardView}>
                 <MappingCard data={entMaps.artifacts}
                     entityName={entMaps.entityType}
+                    getMappingArtifactByMapName={getMappingArtifactByMapName}
                     deleteMappingArtifact={deleteMappingArtifact}
                     createMappingArtifact={createMappingArtifact}
-                    canReadWrite={canReadWrite}
-                    canReadOnly={canReadOnly} />
+                    updateMappingArtifact={updateMappingArtifact}
+                    canReadWrite={props.canReadWrite}
+                    canReadOnly={props.canReadOnly}
+                    entitiesInfo={props.entitiesInfo[entMaps.entityType]} />
```

Comment:
What does "entities info" refer to? I'm guessing it's the entityType? If so, then I think this should be "entityType={props.entityTypes[entMaps.entityType]}". 

But I think "entMaps" should be renamed as well to something that's more clear. Based on its usage - it has "entityType" and "artifacts" as properties - I'm guessing it's a collection of things pertaining to an entityType. Perhaps "entityCardData" would be better here, as it seems specific to the data needed to display an entity-specific card?

Reply:
Done.